beebop/utils.py

```diff
@@ -14,21 +14,29 @@
 
 
 def get_args():
+    """
+    Reads in arguments from file
+    """
     with open("./beebop/resources/args.json") as a:
         args_json = a.read()
     return json.loads(args_json, object_hook=lambda d: SimpleNamespace(**d))
 
 
 def generate_mapping(p_hash, fs):
     """
-    PopPUNK generates one overall .graphml file covering all clusters/
-    components. Furthermore, it generates one .graphml file per component,
-    where the component numbers are arbitrary and do not match poppunk cluster
-    numbers. To find the right component file by cluster number, we need to
-    generate a mapping to be able to return the right component number based
-    on cluster number. This function will generate that mapping by looking up
-    the first filename from each component file in the csv file that holds all
-    filenames and their corresponding clusters.
+    PopPUNKs network visualisation generates one overall .graphml file
```

Comment:
Another form of docs that might be useful to add is a little diagram of the flow of data through the program, which would make this easier to understand

Reply:
Something like this: https://whimsical.com/beebop-CvCKccrmsadvV9aBYi4kX4? (This is out of date, but I can update that if it's useful)